tests/typing_tests/cupy_tests/creation_tests/test_basic.pyi

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import cupy as cp
+import cupy as xp
```

Comment:
I felt difficult distinguishing `xp`, `cp`, `np` in test cases 😅 
How about this?

```suggestion
import cupy as xp

import numpy as _np
import cupy as _cp
```

Reply:
I agree about not using `np` and `cp`, but I prefer `import numpy` and `import cupy` without aliasing.

Reply:
Sounds nice!